src/features/vars-schema/env-vars.ts

```diff
@@ -56,7 +56,8 @@ export const BLOCKED_NAMES = addReservedPrefixes([
 	'HOST_LOG_TO_DISPLAY',
 ]);
 
-export const INVALID_CHARACTER_REGEX = /^\d|\W/;
+const INVALID_ALPHANUMERIC_REGEX = /^\d|\W/;
+export const INVALID_CHARACTER_REGEX = /^[\d:]|[^A-Za-z0-9_:]/;
```

Comment:
Is this going to mess up the UI @thgreasi since it uses it via the `/config/vars` `invalidRegex` property

Reply:
Oh Indeed @Page- we need to export the new regex as well and update `/config/vars` to preserve the old key and add a new to define the config var regex :facepalm:

Reply:
I don't understand, `/config/vars` validates the configuration variable format no? Then I think the export is ok. Moreover, I tested the UI with this new OB version and without this export the configuration menu failed to set the right values